lms/envs/common.py

```diff
@@ -5544,3 +5544,10 @@ def _should_send_learning_badge_events(settings):
 # .. setting_default: empty dictionary
 # .. setting_description: Dictionary with additional information that you want to share in the report.
 SURVEY_REPORT_EXTRA_DATA = {}
+
+
+# .. setting_name: DISABLED_COUNTRIES_FROM_USER_REGISTRATION_FORM
+# .. setting_default: []
+# .. setting_description: List of country codes that should be disabled in the user registration form.
+# .. eg ['US', 'CA']
+DISABLED_COUNTRIES_IN_USER_REGISTRATION_FORM = []
```

Comment:
(I know that this is still in draft, so I'm sorry for leaving feedback early.)

I absolutely love addressing the issue this way. My only question is whether  it makes sense for the environment variable to be specifically `DISABLED_COUNTRIES_IN_USER_REGISTRATION_FORM` instead of `DISABLED_COUNTRIES`. I'm sure that there is some use case you already know about where we might want to disable a country from the country list in user registration only, and not everywhere else. But on the other hand, I already know that there is a desire to be able to disable a country consistently across the user registration form and the account MFE, so I'm thinking of something more general.

 I know the question of too much-abstraction versus too much specificity is never entirely solvable, but I just figured I would raise it as a question.